Index: src/Components/UserForm/UserForm.jsx
===================================================================
diff --git a/src/Components/UserForm/UserForm.jsx b/src/Components/UserForm/UserForm.jsx
deleted file mode 100644
--- a/src/Components/UserForm/UserForm.jsx	
+++ /dev/null	
@@ -1,37 +0,0 @@
-import {Button} from "react-bootstrap";
-import {useState} from "react";
-
-const defaultFormValue = {
-    firstName: "",
-    lastName: "",
-    age: "",
-}
-export const UserForm = () => {
-    const [formValue, setFormValue] = useState(defaultFormValue);
-    const inputChangeValue = ({target: {value} => console.log(value)})
-    return (
-        <form>
-            <div className='mb-3'>
-                <label>First Name
-                    <input
-                        onChange={(event) => inputChangeValue(event, 'firstName')}
-                        value={formValue.firstName}
-                        type='text'
-                        placeholder="enter me"/>
-                </label>
-            </div>
-            {/*<div className='mb-3'>*/}
-            {/*    <label>Last Name*/}
-            {/*        <input value={formValue.lastName} type='text' placeholder="enter me"/>*/}
-            {/*    </label>*/}
-            {/*</div>*/}
-            {/*<div className='mb-3'>*/}
-            {/*    <label>Age*/}
-            {/*        <input value={formValue.age} type='number' placeholder="enter me"/>*/}
-            {/*    </label>*/}
-            {/*</div>*/}
-
-            <Button>Save</Button>
-        </form>
-    )
-}
\ No newline at end of file
